plugins/com.google.cloud.tools.eclipse.sdk/src/com/google/cloud/tools/eclipse/sdk/internal/CloudSdkInstallJob.java

```diff
@@ -53,7 +55,8 @@ public CloudSdkInstallJob(MessageConsoleStream consoleStream, ReadWriteLock clou
    * context (e.g., that deployment fails as the Cloud SDK could not be installed).
    */
   @Override
-  protected IStatus modifySdk(IProgressMonitor monitor) {
+  protected IStatus modifySdk(IProgressMonitor monitor)
```

Comment:
See my earlier comment. Let's do what you did in `CloudSdkUpdateJob`, to be consistent.